src/lib.rs

```diff
@@ -190,14 +191,16 @@ pub extern "C" fn GA_connect(
     let network_name = read_str(network_name);
 
     let network = tryit!(Network::get(&network_name).or_err("missing network"));
-    let rpc = tryit!(network.connect());
-    let wallet = Wallet::new(rpc);
+    let wallet = tryit!(Wallet::new(network));
 
     log::set_max_level(log_filter(log_level));
 
     sess.network = Some(network_name);
     sess.wallet = Some(wallet);
 
+    // give the first tick
+    tryit!(sess.tick());
```

Comment:
This currently happens only after the user logs in (via `GA_session::hello()`). This [was an intentional choice](https://github.com/Blockstream/gdk_rpc/commit/f1a8ef86cd508c230d5f9fd57103fdfec77f7a2b), but I don't quite remember why :o

Reply:
I ran into troubles there for the fee estimation test, I think. The test would find empty data because there would not have been a tick. Only non-invasive solution was to wait 5 seconds, but the entire test suite is less than 5 seconds, so that's a bit annoying.